app/letsencrypt_service

```diff
@@ -126,9 +126,10 @@ function update_certs {
     unset LETSENCRYPT_CONTAINERS
     # shellcheck source=/dev/null
     source /app/letsencrypt_service_data
-
+    
```

Comment:
Whitespace issues ?

Reply:
Probably a side effect of "automatic indentation" of my Editor.